### PR TITLE
Show descriptions for specifications as popups. Fixes #236

### LINKED ISSUE #236: Add description pop-up tooltips over examples strings on the web page
I received a feedback that it would be cool to have tooltips showing over our Spec examples names on the web page. We already have these strings in our API, e.g.:

```
    "descriptions": {
        "OPEN_API": "Open API - Generate OpenAPI-compliant API documentation for your microservices",
        "CONFIG": "Configuration - externalize and manage your configuration parameters outside your microservices",
        "HEALTH_CHECKS": "Health Checks - Verify the health of your microservices with custom verifications",
        "REST_CLIENT": "Rest Client - Invoke RESTful services in a type-safe manner",
        "FAULT_TOLERANCE": "Fault Tolerance - all about bulkheads, timeouts, circuit breakers, retries, etc. for your microservices",
        "JWT_AUTH": "JWT Propagation - propagate security across your microservices",
        "OPEN_TRACING": "Open Tracing - trace the flow of requests as they traverse your microservices",
        "METRICS": "Metrics - Gather and create operational and business measurements for your microservices"
    }
```

What we probably need is to pull these into the web page gui...

### PR DESCRIPTION
**Pull request by rdebusscher**

### CODE REVIEW

**Comment by rdebusscher** [comment]
As draft since it is for 1.4

**Comment by rdebusscher** [comment]
Specifications looks now like this

MPStarter_spec_new_layout

**Comment by Emily-Jiang** [comment]
ah. I have one comments on Health Checks. Can you update Health Checks to Health as the spec is called Health.

File: src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/model/MicroprofileSpec.java [hunk 1]
```diff
@@ -31,41 +31,49 @@
...
                     MicroProfileVersion.MP20, MicroProfileVersion.MP21, MicroProfileVersion.MP22,
                     MicroProfileVersion.MP30, MicroProfileVersion.MP32, MicroProfileVersion.MP33))
     , METRICS("metrics", "Metrics",
             "Metrics - Gather and create operational and business measurements for your microservices",
+            "https://github.com/eclipse/microprofile-metrics/releases/tag/%s",
             Arrays.asList(MicroProfileVersion.MP12, MicroProfileVersion.MP13, MicroProfileVersion.MP14,
                     MicroProfileVersion.MP20, MicroProfileVersion.MP21, MicroProfileVersion.MP22,
                     MicroProfileVersion.MP30, MicroProfileVersion.MP32, MicroProfileVersion.MP33))
     , HEALTH_CHECKS("health_checks", "Health Checks",
             "Health Checks - Verify the health of your microservices with custom verifications",
```
Comment (Emily-Jiang) [inline src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/model/MicroprofileSpec.java:57]: Health Checks -> Health

File: src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/model/MicroprofileSpec.java [hunk 2]
```diff
@@ -31,41 +31,49 @@
...
             Arrays.asList(MicroProfileVersion.MP12, MicroProfileVersion.MP13, MicroProfileVersion.MP14,
                     MicroProfileVersion.MP20, MicroProfileVersion.MP21, MicroProfileVersion.MP22,
                     MicroProfileVersion.MP30, MicroProfileVersion.MP32, MicroProfileVersion.MP33))
     , METRICS("metrics", "Metrics",
             "Metrics - Gather and create operational and business measurements for your microservices",
+            "https://github.com/eclipse/microprofile-metrics/releases/tag/%s",
             Arrays.asList(MicroProfileVersion.MP12, MicroProfileVersion.MP13, MicroProfileVersion.MP14,
                     MicroProfileVersion.MP20, MicroProfileVersion.MP21, MicroProfileVersion.MP22,
                     MicroProfileVersion.MP30, MicroProfileVersion.MP32, MicroProfileVersion.MP33))
     , HEALTH_CHECKS("health_checks", "Health Checks",
```
Comment (Emily-Jiang) [inline src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/model/MicroprofileSpec.java:56]: Health Checks -> Health

**Comment by Emily-Jiang** [comment]
@rdebusscher is going to propose a new PR to fix the wording for Health Check -> Health.

**Pull request merged**